x-pack/plugin/ml/qa/native-multi-node-tests/build.gradle

```diff
@@ -27,8 +27,6 @@ task copyKeyCerts(type: Copy) {
 sourceSets.test.resources.srcDir(keystoreDir)
 processTestResources.dependsOn(copyKeyCerts)
 
-// Disabled and tracked here https://github.com/elastic/elasticsearch/issues/45405
-integTest.enabled = false
 testingConventions.enabled = false
```

Comment:
I think the `testingConventions.enabled = false` should also be deleted.  It was only added because disabling the integration tests resulted in dead test source files.